idl/cherami_server/shared.thrift

```diff
@@ -379,3 +392,8 @@ struct ListExtentsStatsResult {
   1: optional list<ExtentStats> extentStatsList
  10: optional binary nextPageToken
 }
+
+struct SealExtentRequest {
+  1: optional string extentUUID
+  2: optional i64 (js.type = "Long") sequenceNumber
```

Comment:
ack.. yeah i was actually just about to update the diff with a newer diff with this name, since moving the SealExtent to "shared.thrift" caused my server diff to unnecessarily bloat with bunch of import changes on replicator/store/controller..